src/user.py

```diff
@@ -50,19 +50,21 @@ def reputation(self):
         """
         return {**self._reputation_hist, 'reputation_now': self._reputation_now}
 
+
     def commit_user_to_DB(self, user):
         try:
             session.add(user)
             session.commit()
 
-        except exc.IntegrityError:
+        except exc.IntegrityError as e:
+            logger.error(e)
             session.rollback()
-            # log the problem in user logger, Can't print user name with spacial/ unknown characters
-            logger.warning(f'IntegrityError: "Duplicate entry for key users.name ranked')
-
-            # insert user information to not scrapped users logger, Can't print user name with unknown characters
-            logger.error(f'IntegrityError: "Duplicate entry for key users.name ranked {self._rank} at'
-                                      f' website {self._website_name} with url: {self._url}" ')
+            # # log the problem in user logger, Can't print user name with spacial/ unknown characters
+            # logger.warning(f'IntegrityError: "Duplicate entry for key users.name ranked')
+            #
+            # # insert user information to not scrapped users logger, Can't print user name with unknown characters
+            # logger.error(f'IntegrityError: "Duplicate entry for key users.name ranked {self._rank} at'
+            #                           f' website {self._website_name} with url: {self._url}" ')
```

Comment:
Are we also logging the error?
next time when having an issue we would like to know what caused it

Reply:
Yes, we do it in  line 60 -maybe you can modify the logger to print the error and some adjusted string

Reply:
Yes, we do it in line 60 -maybe you can modify the logger to print the error and some adjusted string